circe-yaml/src/main/scala/io/circe/yaml/Parser.scala

```diff
@@ -96,10 +96,8 @@ final case class Parser(
 }
 
 object Parser {
-  val defaultMaxAliasesForCollections: Int = 50 // to prevent YAML at
   // https://en.wikipedia.org/wiki/Billion_laughs_attack
   val defaultNestingDepthLimit: Int = 50
-  val defaultCodePointLimit: Int = 3 * 1024 * 1024 // 3MB
```

Comment:
This is a breaking change. Maybe we can use `def` to point to the new location with a `@deprecated` annotation?

Reply:
Is it? MiMa seems to be happy. Wasn't this code merged only after last release of circe-yaml?
I don't mind doing it the way you suggest, but maybe we don't need to worry about my change, since it's not actually breaking anything.